java/component/keyple-core/src/main/java/org/eclipse/keyple/core/command/AbstractIso7816CommandBuilder.java

```diff
@@ -26,14 +26,17 @@
  *   <li>the built APDURequest,
  *   <li>the corresponding AbstractApduResponseParser class.
  * </ul>
+ *
+ * @since 0.9
  */
 public abstract class AbstractIso7816CommandBuilder extends AbstractApduCommandBuilder {
 
   /**
    * Abstract constructor to build a command with a command reference and an {@link ApduRequest}.
    *
-   * @param commandReference command reference
-   * @param request ApduRequest
+   * @param commandReference The {@link SeCommand} used as Command reference (should be not null)
+   * @param request The {@link ApduRequest} used for the request (should not be null)
```

Comment:
request can be null. The "command builder" objects derived from this class in Keyple-Calypso are often built with a call to super with request = null...